cmd/buildkite-mcp-server/main.go

```diff
@@ -44,13 +47,19 @@ func main() {
 
 	logger := zerolog.New(os.Stderr).With().Timestamp().Logger()
 
+	// are we in an interactive terminal?
+	if isatty.IsTerminal(os.Stdout.Fd()) {
+		logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr, FormatTimestamp: func(i any) string {
+			return time.Now().Format(time.Stamp)
+		}}).With().Caller().Logger()
+	}
+
 	if cli.Debug {
 		logger = logger.Level(zerolog.DebugLevel).With().Caller().Logger()
 	}
 	log.Logger = logger
-	zerolog.DefaultContextLogger = &logger
```

Comment:
This is needed as a base logger when using `log.Ctx(ctx)` btw if you don't manually add one into the context, which we haven't been doing

Reply:
@catkins I am relying on code which sets the context to pass the logger through now, however I need to review the HTTP server to verify equivelant is present.